src/scenes/home/branding/branding.js

```diff
@@ -0,0 +1,29 @@
+import React, { Component } from 'react';
+import Section from 'shared/components/section/section';
+import BrandingLogo from './logo/logo';
+import BrandingColor from './color/color';
+import styles from './branding.css';
+
+class Branding extends Component {
```

Comment:
Since you're not utilizing state or life cycle methods in this component, you can convert it into a functional component for an eventual performance boost when we get the repo to React 16.

Like so...
```
const Branding = () => (
  <div className={styles.brandingConatiner}>
    <BrandingLogo />
    <BrandingColor />
    <Section title="Fonts" theme="grey">
      <div className={styles.brandingFonts}>
        <h6>PF DIN DISPLAY PRO REGULAR</h6>
        <img src="./images/fonts/pf-din-font.png" alt="PF DIN REGULAR FONT" />
        <p><a href="https://ocbranding.squarespace.com/s/PFDin.zip">Download PF Din font files</a></p>
        <p>&nbsp;</p>
        <h6>NOTO SERIF REGULAR</h6>
        <img src="./images/fonts/noto-serif-font.png" alt="NOTO SERIF REGULAR FONT" />
        <p><a href="https://ocbranding.squarespace.com/s/NotoSerif.zip">Download Noto Serif font files</a></p>
      </div>
    </Section>
  </div>
)
```